react-demo/src/App.js

```diff
@@ -51,6 +51,20 @@ function App() {
 		}
 	}, [fluidMap]);
 
+	React.useEffect(() => {
+		let container;
+		(async () => {
+		  container = await getContainer();
+		  const myMap = container?.initialObjects?.myMap;
+		  setFluidMap(myMap)
+		})();
+	  
+		// Return a de-registration function that disconnects the old container whenever the hook re-runs
+		return () => {
+		  if (container != null) container?.disconnect();
```

Comment:
```suggestion
		  if (container != undefined) container?.disconnect();
```

I think you want to check `undefined` here, not `null`.

Reply:
Checking `container == null` returns `true` for both `undefined` and `null`, but checking `container === undefined` can be a bit more explicit in what we are testing for.